pyccel/ast/core.py

```diff
@@ -268,6 +268,9 @@ def name(self):
     def target(self):
         return self._target
 
+    def __repr__(self):
+        return '{0} as {1}'.format(str(self.name), str(self.target))
```

Comment:
can we have a better desc such as `rename old to new`?

Reply:
So you would expect the object `Import(source, [AsName(old,new)])` to be printed as something like:
```
Import(from source import rename old to new)
```
?
Would this be better for a `repr`:
```
AsName(old as new)
```
?
I am not sure `rename old to new` makes the use case clear

Reply:
Ah, You are right, it's more readable than my example.